shell/gpu/gpu_surface_gl.cc

```diff
@@ -139,7 +139,7 @@ static sk_sp<SkSurface> WrapOnscreenSurface(GrDirectContext* context,
   GrBackendRenderTarget render_target(size.width(),     // width
                                       size.height(),    // height
                                       0,                // sample count
```

Comment:
🤔 How is this working with a sample count of 0? Perhaps Skia clamps it to 1? If you're in the mood for a drive by fix, I'd specify 1 here just so its clear.

Reply:
I'm pretty sure Skia does some magic to make sure the value turns out right. We're using 0 in a few places - I'd slightly prefer cleaning that up in a separate patch.

There's also a todo in the context_options file that's tied to a skbug that's resolved that we can try to remove.